### PR TITLE
Ana/fix signout fault

### LINKED ISSUE #3405: Signout fault: upon refreshing last account gets signed in again
**Describe the bug**

When you are logged in, doesn't matter in which network, and you sign out, everything seems to be correct until the moment you refresh the page. Then you are logged in back again to the last account you were signed in with (in that network).

**Expected behavior**

It should never signed you back in by refreshing the page, only when selecting the network from the Networks section.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: 
 - Browser: 
 - Version: 

**Smartphone (please complete the following information):**
 - Device: 
 - OS: 
 - Browser 
 - Version 

**Additional context**

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #3405 

**Description:**

With this PR singout works as you would expect.

Once you sign out, that session is removed from localStorage and to sign back in again you need to go through the sign in process (you are not signed in back again by refreshing the page).

Also, the automatic signin feature when switching networks stays the same, nothing changes here (thanks to a hack).

TODO: test :crying_cat_face: 

Thank you! 🚀

---

For contributor:

- [x] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/vuex/modules/session.js [hunk 1]
```diff
@@ -148,16 +144,23 @@ export default () => {
...
-      dispatch(`resetSessionData`)
+      dispatch(`resetSessionData`, networkId)
       commit(`setSignIn`, false)
     },
-    resetSessionData({ commit, state }) {
+    resetSessionData({ commit, state }, networkId) {
       state.history = ["/"]
       commit(`setUserAddress`, null)
-      localStorage.removeItem(`session`)
+      // Hack. In case we receive the networkId parameter, we reset session data for
```
Comment (faboweb) [inline src/vuex/modules/session.js]: In which case wouldn't we receive the networkId?

Comment (Bitcoinera) [reply src/vuex/modules/session.js]: When called from `connection.js` (while executing `setNetwork`). This is the hack

Comment (faboweb) [reply src/vuex/modules/session.js]: is `session` in localStorage set at any point?

Comment (Bitcoinera) [reply src/vuex/modules/session.js]: I guess during signin

Comment (faboweb) [reply src/vuex/modules/session.js]: Wrooooong ;P

Comment (Bitcoinera) [reply src/vuex/modules/session.js]: :crying_cat_face:

**Pull request merged**